src/administration/users.md

```diff
@@ -39,26 +39,22 @@ If you want your users to be able to see everything (Reader), but only commit to
 
 ## Manage user permissions at the project level
 
-Access the project-level configuration.
+From your list of projects, enter into the project where you want to view or edit user permissions. At this point, you will not have entered a particular environment. Click the Settings tab at the top of the page, the click the ‘Access’ tab on the left to show the project-level users and their roles. 
```

Comment:
"entering into a project" or "entering an environment" seems like weird wording here.  I don't have a specific alternative to recommend but this feels like the wrong metaphor to me.

Reply:
selected?

Reply:
That works better, but then the second sentence needs to be revised, too.